src/framework/accessibility/internal/accessibleapprootinterface.cpp

```
@@ -0,0 +1,154 @@
+
+    for (int i = 0; i < m_root->windowCount(); ++i) {
+        if (m_root->windowAt(i) == focused) {
+            QAccessibleInterface* windowIface = child(i);
+            if (windowIface) {
+                QAccessibleInterface* focusedChild = windowIface->focusChild();
+                return focusedChild ? focusedChild : windowIface;
+            }
+        }
+    }
+
+    return nullptr;
```

Cppcheck (2.20.0)

[style] The function 'toMap' is never used.

(unusedFunction)